### PR TITLE
docs(commands/bump.md): document --yes option for cz bump

### LINKED ISSUE #1698: `cz bump --yes` is not documented
### Type

- [ ] Content inaccurate
- [x] Content missing
- [ ] Typo

### URL

https://commitizen-tools.github.io/commitizen/commands/check/

### Description

Also, the description in CLI is "accept automatically questions done". Very unclear to me.

### PR DESCRIPTION
**Pull request by Stoiicc**
## Description
This PR documents the --yes option for the cz bump command, which is currently shown in the CLI help output but not explained in the documentation. The new section clarifies its usage.

## Checklist

- [x] I have read the [contributing guidelines](https://commitizen-tools.github.io/commitizen/contributing/)

### Code Changes

- [ ] Add test cases to all the changes you introduce
- [ ] Run `uv run poe all` locally to ensure this change passes linter check and tests
- [ ] Manually test the changes:
  - [x] Verify the feature/bug fix works as expected in real-world scenarios
  - [ ] Test edge cases and error conditions
  - [ ] Ensure backward compatibility is maintained
  - [ ] Document any manual testing steps performed
- [x] Update the documentation for the changes

### Documentation Changes

- [x] Run `uv run poe doc` locally to ensure the documentation pages renders correctly
- [ ] Check and fix any broken links (internal or external) in the documentation

## Expected Behavior
Users can clearly understand what the --yes option does when running cz bump.

## Additional Context
Fixes #1698

### CODE REVIEW

File: docs/commands/bump.md [hunk 1]
```diff
@@ -507,3 +507,11 @@ Supported variables:
 [pep440]: https://www.python.org/dev/peps/pep-0440/
 [semver]: https://semver.org/
 [version_files]: ../config/bump.md#version_files
+
```
Comment (bearomorphism) [inline docs/commands/bump.md]: nit: please keep the links at the end of the file, thanks

**Review by bearomorphism — COMMENTED** [review]
LGTM. One nit

**Comment by bearomorphism** [comment]
@BelpHegoR17 could you rebase your branch to the latest master? We fixed the pipeline failure.

Thanks!

File: docs/commands/bump.md (hunk 1 above)
Comment (Stoiicc) [reply docs/commands/bump.md]: Fixed, thanks
